client/components/feature-modules/blocks/components/panel/panel-wrapper.tsx

```
@@ -149,6 +199,57 @@ export const PanelWrapper: FC<Props> = ({
+            // Activate focused toolbar button with Enter
+            if (event.key === "Enter" && toolbarFocusIndex >= 0) {
+                if (isInput) return;
+                event.preventDefault();
+
+                // Determine which menu should be opened
+                let buttonIndex = 0;
+                let targetMenu: "quick" | "insert" | "details" | "actions" | null = null;
+
+                if (toolbarFocusIndex === buttonIndex++) {
+                    targetMenu = "quick";
+                } else if (allowInsert && toolbarFocusIndex === buttonIndex++) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)